self-tests.yml

```diff
@@ -708,6 +708,8 @@ tasks:
             SWAGGER_JSON_FILE: docs/swagger.json
             SWAGGER_OLD_SUM_FILE: docs/swagger.old.sum
             OUTPUT_SUM_FILE: docs/swagger.sum
+            environment: ${environment}
+            environment-suffix: ${environment-suffix}
```

Comment:
I think we should set the suffix for prod / our prod Evergreen project too to something. This is prone to being overriden with staging project clones that don't have the latest variables (e.g. there's a couple clones and copies of our projects, if any of them or new ones don't have the variable set, it will overwrite prod)

Reply:
See my other comment.